src/app/content/highlights/hooks/utils.ts

```diff
@@ -4,12 +4,22 @@ import {
   Highlight,
   Highlights
 } from '@openstax/highlighter/dist/api';
-import { AppServices } from '../../../types';
+import omit from 'lodash/fp/omit';
+import { AppServices, AppState, MiddlewareAPI, Store } from '../../../types';
 import { assertDefined } from '../../../utils';
+import { book as bookSelector } from '../../selectors';
 import { Book } from '../../types';
 import { stripIdVersion } from '../../utils';
+import * as select from '../selectors';
 import { HighlightLocationFilters, SummaryHighlightsPagination, } from '../types';
 import { addSummaryHighlight, getHighlightLocationFilterForPage } from '../utils';
+import { getNextPageSources } from '../utils/paginationUtils';
+
+type fetchFunctionBody = (args: { previousPagination: SummaryHighlightsPagination, getState: Store['getState'],
+    highlightClient: AppServices['highlightClient'],
+    highlights?: Highlight[]
+    sourcesFetched: string[]}
+) =>  Promise<{pagination: SummaryHighlightsPagination, highlights: Highlight[]}>;
```

Comment:
Indentation + double spae before `Promise` - btw. if you are using vscode you can sett default indentation for a given project and auto fixing on `ctrl + s`